.github/workflows/parallal-staging.yaml

```
@@ -0,0 +1,105 @@
+jobs:
+  deploy:
+    name: White-label production deploy
+    runs-on: ubuntu-latest
+    env:
+      SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+      DEPLOYMENTS_CONFIG_v2: ${{ secrets.DEPLOYMENTS_CONFIG_v2 || vars.DEPLOYMENTS_CONFIG_v2 }}
+      VERCEL_TOKEN_FB: ${{ secrets.VERCEL_TOKEN }}
+      VERCEL_ORG_ID_FB: ${{ secrets.VERCEL_ORG_ID }}
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)